From 16eb3d1859d3c6fbfb1407aa68574c77ff68629d Mon Sep 17 00:00:00 2001
From: Demon000 <demonsingur@gmail.com>
Date: Wed, 10 Jan 2018 20:52:45 +0200
Subject: [PATCH 03/16] sepolicy: allow system_server to read alarm boot prop

Fixes the following denial.

avc: denied { read } for comm="system_server"
name="u:object_r:alarm_boot_prop:s0"
dev="tmpfs"
ino=10769
scontext=u:r:system_server:s0
tcontext=u:object_r:alarm_boot_prop:s0
tclass=file

Change-Id: I142c92d9785bd79fba7d50e70ab4d689bb2ba7d5
---
 vendor/common/system_server.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/vendor/common/system_server.te b/vendor/common/system_server.te
index 2f40d7e4..a33290e6 100644
--- a/vendor/common/system_server.te
+++ b/vendor/common/system_server.te
@@ -133,6 +133,7 @@ allow system_server fingerprintd_data_file:dir { rw_dir_perms rmdir };
 #for Wifi module this is needed
 allow system_server system_file:system module_load;
 
+get_prop(system_server, vendor_alarm_boot_prop)
 userdebug_or_eng(`
   diag_use(system_server)
 ')
-- 
2.17.1

